src/AutoMapper/PropertyMap.cs

```diff
@@ -41,7 +41,7 @@ public PropertyMap(PropertyMap inheritedMappedProperty, TypeMap typeMap)
         public LambdaExpression CustomExpression { get; private set; }
         public MemberInfo CustomSourceMember { get; set; }
         public bool UseDestinationValue { get; set; }
-        public bool ExplicitExpansion { get; set; }
+        public bool ExplicitExpansion { get { return _explicitExpansion.GetValueOrDefault(); } set { _explicitExpansion = value; } } private bool? _explicitExpansion; 
```

Comment:
Well I really just wanted my original question answered - should ExplicitExpansion just be nullable, and remove that backing field.